src/components/NavBar/NavBar.js

```diff
@@ -53,11 +53,9 @@ const NavBar = () => {
               </NavLink>
             </Menu.Item> */}
 
-            <Menu.Item key="6">
```

Comment:
Good move removing from the nav, eliminates unnecessary underline

Reply:
I agree, great work.